native/dlarfb.go

```diff
@@ -0,0 +1,422 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+import (
+	"github.com/gonum/blas"
+	"github.com/gonum/blas/blas64"
+	"github.com/gonum/lapack"
+)
+
+// Dlarfb applies a block reflector to a matrix.
+//
+// In the call to Dlarfb, c is multiplied by the implicitly defined matrix h.
```

Comment:
What about modifying it to: `... the mxn matrix c is ... matrix h as follows:` ? Otherwise the size of c is not explicitly stated and this seems like a good place.